deli_counter.rb

```diff
@@ -0,0 +1,46 @@
+# Anshul Arora
+katz_deli = []
+$current_line = []
+
+def line(arr)
+  if arr.empty?
+    puts "The line is currently empty." 
+  else
+    $current_line.clear
+    line_helper(arr)
+    puts "The line is currently:#{$current_line.join}"
```

Comment:
we will work on this, this isn't a great way to build a string for output since the logic is happening in another method (line 19)